src/core/public/chrome/ui/header/collapsible_nav.tsx

```diff
@@ -95,7 +100,14 @@ export function CollapsibleNav({
   button,
   ...observables
 }: Props) {
-  const navLinks = useObservable(observables.navLinks$, []).filter((link) => !link.hidden);
+  const navLinks = useObservable(observables.navLinks$, []).filter(
+    // Filterting out hidden links and the integrations one in favor of a specific Add Data button at the bottom
+    (link) => !link.hidden && link.id !== 'integrations'
+  );
```

Comment:
NIT: Be aware that the `filter` is not part of the effect, and therefor will be re-executed on each rendering.

Something more correct would probably be:

```ts
const navLinks = useObservable(observables.navLinks$, [])
const nonIntegrationLinks = useMemo(() => navLinks.filter(...), [navLinks]);
const integrationLink = useMemo(() => navLinks.find(...), [navLinks]);
```